EnglishTranslation-master/QUEST_LV_0300.tsv

```diff
@@ -1206,30 +1206,30 @@ QUEST_LV_0300_20160310_001205	What do we do with the useless scriptures?{nl}We u
-QUEST_LV_0300_20160310_001227
-QUEST_LV_0300_20160310_001228
-QUEST_LV_0300_20160310_001229
-QUEST_LV_0300_20160310_001230
-QUEST_LV_0300_20160310_001231
-QUEST_LV_0300_20160310_001232
+QUEST_LV_0300_20160310_001209 Villager James
+QUEST_LV_0300_20160310_001210 Oh, you're the newcomer. {nl}Pleasure to meet you. {nl}
+QUEST_LV_0300_20160310_001211 I'm here to cleanse the Goddess' strength using the crimson blood of demons.{nl}However it seems I've run out.{nl}
+QUEST_LV_0300_20160310_001212 Normally Dominikas gets it for me but he seems to have be busy at the moment.{nl}Not to mention the fact that we are currently under their protection so we can't attack them either.{nl}
+QUEST_LV_0300_20160310_001213 Thats why I want you to go get some crimson demon blood.{nl}As a recruit you can do this much, no?
+QUEST_LV_0300_20160310_001214 Thanks.{nl}Demon blood has the peculiar power to cleanse the Goddess' strength.{nl} 
+QUEST_LV_0300_20160310_001215 Look around you, all of this was the result of my purification.{nl}How about it, beautiful isn't it?{nl}
+QUEST_LV_0300_20160310_001216 Well it's about time I leave.{nl}You're free to wander the monestary but try not to go near the black wall in the upper annex research facility.{nl}
+QUEST_LV_0300_20160310_001217 That wall is full of holy energy so that only those chosen by the Divine Tree such as Dominikas can go there.{nl}Well then, see you later.
+QUEST_LV_0300_20160310_001218 ..How did you get in here?{nl}This is a holy place that only those who are chosen by the Divine Tree may enter.{nl}
+QUEST_LV_0300_20160310_001219 Oh my, it appears that the Divine Tree has already chosen you.{nl}I knew you were something special the moment I laid my eyes upon you.{nl}
+QUEST_LV_0300_20160310_001220 To think that you were chosen the moment you joined, this is quite an amazing turn of events.{nl}There is a very important ritual coming up.. a ritual to bestow the Divine Tree's blessings.{nl}
+QUEST_LV_0300_20160310_001221 All the other followers are waiting at the passage leading to the annex.{nl}Although I must go ahead in order to prepare for the ritual please take your time.
+QUEST_LV_0300_20160310_001222 Hurry and deliver this.{nl}I can no longer stand to see the Goddess' strength being tainted.
+QUEST_LV_0300_20160310_001223 I feel an evil presence coming from the vines that Dominikas left behind.{nl}To think that our monnestary might have been covered in such a thing, how frightening.{nl}
+QUEST_LV_0300_20160310_001224 Is there any way to get rid of those vines?
+QUEST_LV_0300_20160310_001225 Hmm..You say that you need a particular reagent to get rid of the vine?{nl}
+QUEST_LV_0300_20160310_001226 You've arrived at just the right time.{nl}The reagent that Vaidotas was working on is done.
+QUEST_LV_0300_20160310_001227 Hah!{nl}I'll call it Lucien's Special Reagent!{nl}
+QUEST_LV_0300_20160310_001228 ..What's wrong with your face?{nl}It's real this time! I made sure it's safe this time!{nl}
```

Comment:
ellipsis (...)

`What's wrong with your face` - Um, it refers more to the facial expression. Maybe: `What's with that look on your face?`